.github/workflows/go_build.yaml

```diff
@@ -68,4 +68,93 @@ jobs:
 
     - name: Spec Test Coverage
       run: |
-            go run tests/spectests/cmd/analyzer/analyzer.go
\ No newline at end of file
+            # Output the full report to console.
+            go run tests/spectests/cmd/analyzer/analyzer.go
+
+
+  go_spec_test_badge_update:
```

Comment:
future todo: may also be cool to print the coverage summary as a comment on future PRs, which should be fairly easy https://github.com/actions/github-script#comment-on-an-issue